components/navbar/Navbar.js

```diff
@@ -145,16 +145,19 @@ export default function Navbar() {
                   />
                 </Link>
               </div>
-              <div className="hidden md:block">
+             <div className="hidden md:block">
                 <div className="ml-10 flex items-baseline space-x-4">
-                  {primary.map((item) => (
-                    <NavLink
+                  <ul>
+                    {primary.map((item) => (
+                      <NavLink
                       key={item.name}
                       path={router.pathname}
                       item={item}
                       setIsOpen={setIsOpen}
                     />
                   ))}
+                  </ul>
```

Comment:
Please install the Prettier plugin on VScode as the formatting does not look correct

Reply:
The prettier config was surprisingly not working yesterday but stated working now. I think the fix is okay.